### PR TITLE
initial server commit. partial response functionality built

### PR DESCRIPTION
**Pull request by RobbMAlexander**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const weatherData = require('./data/weather.json');
+
+app.get('/', (req, res) => {
+  res.send('Greeting check for root URL');
+});
+
+app.get('/weather', (req, res) => {
+  let city = req.query;
+  res.send(weatherData.city.city_name);
+});
```
Comment (RivaD2) [inline server.js]: Good work so far  on task 1 and starting on task 2 for weather 👍 

- This endpoint will process the request that contains `lat, lon and the searchQuery`
- We will need to use `find()` to discover which city the `lat, lon and searchQuery` info belong to

File: server.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const cors = require('cors');
```
Comment (RivaD2) [inline server.js:5]: Ryan covered more about CORS today in class. 

- We will need to use the middleware. In addition to loading the module, we will need to add: `app.use(cors())` 👍

File: server.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+});
+
+// app.get('/', (req, res) => {
+//   res.send('Greeting check for root URL');
+// });
+
+// app.get('/', (req, res) => {
+//   res.send('Greeting check for root URL');
+// });
+
```
Comment (Hambalieu) [inline server.js:85]: delete commented code

**Review by Hambalieu — COMMENTED** [review]
Robbs server setup looks good

**Pull request merged**